### PR TITLE
[master] KBLA-6: add new kapi for publishing websocker app_event

### PR DESCRIPTION
**Pull request by icehess**
Adding a new KAPI for publishing app_events from blackhole using `publish_app_event` event.

### CODE REVIEW

**Comment by icehess** [comment]
Related PRs:
* https://github.com/2600hz/kazoo-blackhole/pull/14
* https://github.com/2600hz/kazoo-properly/pull/57

File: kazoo_amqp/src/api/kapi_websockets.erl [hunk 1]
```diff
@@ -11,6 +11,11 @@
 
 -export([api_definitions/0, api_definition/1]).
 
+-export([app_event/1
+        ,app_event_v/1
+        ,publish_app_event/1
+        ,publish_app_event/2
+        ]).
```
Comment (lazedo) [inline kazoo_amqp/src/api/kapi_websockets.erl:18]:
```suggestion
-export([event/1
        ,event_v/1
        ,publish/1
        ,publish/2
        ]).
```

in a new `kapi_app`

**Comment by lazedo** [comment]
@icehess this change would great to have in a new `kapi_app` so that any app can use this mechanism, i see a lot of use cases

**Comment by jamesaimonetti** [comment]
JIRA Ticket says closed in favor of diff approach

**Pull request closed without merging**